README.md

```diff
@@ -220,7 +220,11 @@
 
   ```
   yum install ruby
+  yum install ruby-devel
+  yum install zlib-devel
+
   gem install trollop
+  gem install fog
```

Comment:
Thanks!  I forgot I made those changes to get it to work. What about `libxml2-devel` and `libxslt-devel`?  I think something in the fog dependency tre uses nokogiri.